ssl/quic/quic_impl.c

```diff
@@ -1301,14 +1301,7 @@ int ossl_quic_handle_events(SSL *s)
         return 0;
 
     qctx_lock(&ctx);
-    /*
-     * TODO(QUIC SERVER): We should always tick the engine, whether the current
-     * connection is started or not.  When ticking the engine, we MUST avoid
-     * inadvertently starting connections that haven't started, the guards
-     * don't belong here.
-     */
-    if (ctx.qc == NULL || ctx.qc->started)
-        ossl_quic_reactor_tick(ossl_quic_obj_get0_reactor(ctx.obj), 0);
+    ossl_quic_reactor_tick(ossl_quic_obj_get0_reactor(ctx.obj), 0);
```

Comment:
Yes - because they are no longer needed. The quic server branch handles this in a different way (with a check whether the channel has started in the `ossl_quic_channel_subtick` function, which does not exist in the master branch).